src/Subscriptions/Repositories/SubscriptionRepository.php

```diff
@@ -204,28 +189,23 @@ public function update(Subscription $subscription)
 
         DB::query('COMMIT');
 
-        $subscriptionId = DB::last_insert_id();
-
-        $subscription = $this->getById($subscriptionId);
-
-        Hooks::doAction('give_subscription_updated', $subscription);
-
-        return $subscription;
```

Comment:
Do we need to update the `save()` method in the model as well?

Reply:
Nope! that was already done strangely enough.